src/main_server_template.mjml

```diff
@@ -1,14 +1,31 @@
 <mjml>
   <mj-head>
     <mj-attributes>
-      <mj-all font-family="Helvetica" font-size="16px" line-height="20px"/>
+      <mj-all font-family="Helvetica" font-size="16px" />
     </mj-attributes>
+    <mj-style inline="inline">
+      h1 {
```

Comment:
The actual font sizes for headings are resolved by the client, so they could theoretically be different across them. This is why I'm explicitly setting the font sizes here.